site/src/pages/OrganizationSettingsPage/OrganizationProvisionerJobsPage/JobRow.tsx

```diff
@@ -120,7 +120,7 @@ export const JobRow: FC<JobRowProps> = ({ job, defaultIsOpen = false }) => {
 								<>
 									<dt>Completed by provisioner:</dt>
 									<dd className="flex items-center gap-2">
-										<span>{job.worker_id}</span>
+										<span>{job.worker_name}</span>
```

Comment:
Worker name could be empty here if the job is old enough and the provisioner daemon has been cleaned from the DB. We could have some kind of fallback like `[removed]`.

Reply:
Good point! I’ll update it to include a fallback. That said, this raises a question, we still show the “View provisioner” button, which links to the provisioner daemon’s page. If the daemon has been cleaned from the DB, we should avoid showing the button, correct?

Reply:
Oh yeah, good catch. Definitely hide/disable the button as well 👍🏻

Reply:
Handled in commit: https://github.com/coder/coder/pull/17877/commits/caed3963e96b4f4c8fb76847bdf5167675eac781
@BrunoQuaresma let me know if this change is ok 🙂